tests/positive/partitions/creation.go

```diff
@@ -89,15 +89,15 @@ func WipeAndCreateNewPartitions() types.Test {
 					"label": "important-data",
 					"number": 1,
 					"size": 65536,
-					"typeGuid": "B921B045-1DF0-41C3-AF44-4C6F280D3FAE",
-					"guid": "8A7A6E26-5E8F-4CCA-A654-46215D4696AC"
+					"typeGuid": "$uuid0",
+					"guid": "$uuid1"
```

Comment:
see comment at the top of the test. This needs to be different than the input uuid